### ISSUE TITLE
Meltano UI environment variables passed to plugins can cause conflicts

### ISSUE THREAD

**Issue by rabidaudio** [meltano/meltano#6017]
We run the dagit UI through `meltano invoke` using this config:

```yml
  utilities:
    commands:
      scheduler:
        args: run -w $DAGSTER_HOME/workspace.yaml
        executable: dagster-daemon
      ui:
        args: -w $DAGSTER_HOME/workspace.yaml -h 0.0.0.0
        executable: dagit
```

At some point, `meltano invoke dagster:ui` started exiting shortly after the server started. We've been working around it by invoking dagit directly instead (`.meltano/utilities/dagster/venv/bin/dagit -w $DAGSTER_HOME/workspace.yaml -h 0.0.0.0`).

I finally got around to digging into the issue. Turns out the problem is the environment variables `WEB_CONCURRENCY=4 WORKERS=4` being passed to dagit caused it to exit after starting. Setting `MELTANO_UI_WORKERS=1` caused the issue to go away.

AFAICT:

- `MELTANO_UI_WORKERS` is namespaced, but can also be set using non-namespaced variable aliases `WEB_CONCURRENCY` and `WORKERS`
- When invoking plugins, all aliases for all configs are generated and passed to plugins
- These un-namespaced vars can cause conflicts with plugins. Others I noticed that could cause problems: `PORT` and `FORWARDED_ALLOW_IPS`

We could solve this either by not sending any of the meltano ui related config env vars to plugin invocation, or not generate these aliases when generating the project env var configuration (that is, `WEB_CONCURRENCY=4 meltano ui` would be accepted in place of `MELTANO_UI_WORKERS=4` but by renaming it to `MELTANO_UI_WORKERS` before generating the project config).

**Comment by aaronsteers** [comment]
@rabidaudio - Thanks very much for raising this issue.

We're getting a lot more deliberate regarding how environment variables are passed around, and we're removing the car majority of "non-namespaced" variable declarations that were previously implemented using `env_aliases` feature, which is planned to be retired in 2.0.

Bottom line: I expect this will "just work" in 2.0 when we launch next week. Feedback on this or any other changes would be much appreciated if you're able to provide it.

Thanks 🙏

**Comment by rabidaudio** [comment]
Confirmed this is fixed in 2.0 🎉
